public/table-sort.js

```diff
@@ -167,12 +167,29 @@ function tableSortJs(testingTableSortJS = false, domDocumentWindow = document) {
       }
     }
 
-    function getTableData(tableRows, columnData, isFileSize, isDataAttribute) {
+    function getColSpanData(sortableTable, colSpanData, colSpanSum){
+      sortableTable.querySelectorAll("th").forEach(
+        (th,index) => {
+          colSpanData[index] = th.colSpan
```

Comment:
This line looks redundant. You're overwriting `colSpanData[index]` on the next few lines.

Reply:
@strager No, I wasn't overwriting colSpanData, I was making changes to colSpanSum.

Reply:
Oh, you're right. The variable names looked the same to me! =X